src/server/api/user.js

```diff
@@ -10,8 +10,11 @@ import express from "express";
 import * as auth from "../auth";
 import logger from "../logging";
 
+import axiosInstance from "../../common/getAxios";
 import { handler as terrainHandler } from "./terrain";
 
+const WEBHOOK_TEST_BODY = { text: "This is a test message from DE." };
```

Comment:
Minor, but seems like this could be on the client-side and set in the service facade instead of here on the server-side.